vm/systemSmartContracts/esdt.go

```diff
@@ -80,9 +82,16 @@ func (e *esdt) Execute(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 		return vmcommon.UserError
 	}
 
-	switch args.Function {
-	case core.SCDeployInitFunctionName:
+	if args.Function == core.SCDeployInitFunctionName {
 		return e.init(args)
+	}
+
+	if e.disabled {
+		e.eei.AddReturnMessage("ESDT SC disabled")
+		return vmcommon.UserError
+	}
+
+	switch args.Function {
```

Comment:
In governanceContract in the same method Execute, if no one args.Function is found in switch it is returned in the last line: 

	g.eei.AddReturnMessage("invalid method to call")
	return vmcommon.FunctionNotFound

In this method in line 119 it is returned: vmcommon.Ok

This was the intention?

Reply:
thx. fix in another PR as was doing trail of bits audit findings as well.

Reply:
fixed here.